src/decorator/scopes/Scope.ts

```diff
@@ -0,0 +1,12 @@
+import {getMetadataArgsStorage} from "../../";
+import {ScopeMetadataArgs} from "../../metadata-args/ScopeMetadataArgs";
+
+export function Scope(global: boolean = false): Function {
```

Comment:
Boolean parameters are usually considered bad. `{global: boolean}` will be future-proof.